build.gradle.kts

```diff
@@ -58,9 +35,21 @@ kotlin {
     }
 
     cocoapods {
-        summary = "GodTools tool parser"
+        name = "GodToolsShared"
+        summary = "GodTools shared logic"
         license = "MIT"
         homepage = "https://github.com/CruGlobal/kotlin-mpp-godtools-tool-parser"
+        extraSpecAttributes += mapOf(
+            "prepare_command" to """"./gradlew generateDummyFramework"""",
+            "preserve_paths" to """"**/*.*"""",
+        )
+
+        // configure the custom xcode configurations in the godtools-swift project
+        xcodeConfigurationToNativeBuildType += mapOf(
+            "AnalyticsLogging" to NativeBuildType.DEBUG,
+            "Staging" to NativeBuildType.DEBUG,
+            "Production" to NativeBuildType.DEBUG,
```

Comment:
@levieggertcru did I get all custom configurations defined correctly here? the cocoapod support already includes the `Debug` and `Release` configurations.

The difference between `NativeBuildType.DEBUG` and `NativeBuildType.RELEASE` is that `RELEASE` enables code optimizations and bitcode, whereas `DEBUG` doesn't

Reply:
Hey @frett yep that looks good!

Reply:
If we ever define additional configurations, we will need to register them here so that the shared library will work correctly with them.